src/config/interfaces.ts

```diff
@@ -85,15 +85,15 @@ export enum Operators {
   has_group = 'has_group',
   has_phone = 'has_phone',
   has_email = 'has_email',
-  has_value = 'has_value',
   has_district = 'has_district',
   has_state = 'has_state',
   has_ward = 'has_ward',
   has_pattern = 'has_pattern',
   has_error = 'has_error',
   has_intent = 'has_intent',
   has_top_intent = 'has_top_intent',
-  has_category = 'has_category'
+  has_category = 'has_category',
+  has_value = 'has_value' // deprecated, use has_text
```

Comment:
assuming I can't just remove this because there are flows out there using it

Reply:
Unless we add a flow version migration. Remember those?